docs/drafts/derive-serviceaccount.md

```diff
@@ -0,0 +1,578 @@
+# Derive minimal ServiceAccount required for ClusterExtension Installation and Management
+
+OLM v1 does not have permission to install extensions on a cluster by default. In order to install a [supported bundle](../refs/supported-extensions.md), OLM must be provided a ServiceAccount configured with the appropriate permissions. For more information, see the [provided ServiceAccount](./provided-serviceaccount.md) documentation.
+
+This document serves as a guide for how to derive the RBAC necessary to install a bundle.
+
+### Required RBAC
+
+You can determine the specifics of these permissions by referencing the bundle of the ClusterExtension you want to install. The service account must have the following permissions:
+
+* Create, list, watch verbs for all resources that are a part of the install (cannot be scoped to specific resource names).
```

Comment:
The dot points are a bit confusing. The top level says create, list, watch, and the sub-items mentions specific verbs like create (and sometimes manage, which is not clear). If you decide to break this up into two sections as suggested above, then you can break it down to: create, list, watch for cluster-scoped manifest object in the bundle, the CustomResourceDefinitions in the bundle, then separately, the rules in the CSV.  For each, add an explanation.

Reply:
sure

Reply:
this is the final section I need help with